platform-coop-toolkit/coop-library
# feat: use Koko Analytics for sorting by views

## Description

This pull request uses Koko Analytics data for sorting resources by views.

## Steps to test

1. Visit some resources.
2. Sort by view.

**Expected behavior:** Resources are sorted in an order that corresponds to the most viewed resources shown on the Koko Analytics dashboard.

## Additional information

Not applicable.

## Related issues

- Depends on https://github.com/platform-coop-toolkit/coop-library-framework/pull/262.
- Partially resolves #124.
